build-yaml/econet-hvac-esp32.yaml

```diff
@@ -6,10 +6,18 @@ substitutions:
   board: esp32dev
 
 packages:
-  econet-hvac: !include ../econet_hvac.yaml
+  econet:
+    url: https://github.com/esphome-econet/esphome-econet
+    ref: main
+    file: econet_hvac.yaml
   # Uncomment if you have an communicationg Outdoor Unit
   # econet-hvac-odu: !include ../econet_hvac_odu.yaml
```

Comment:
Update this too. Same in the the other file.

Reply:
Unfortunately you can't use substitutions for `packages:` as per https://github.com/esphome/issues/issues/4939

Reply:
I meant update this too to include from the repository URL instead of a relative path.

Reply:
Oh, I see. I thought this was the same comment as above. Good catch! Updated.